frontend/stories/crossingList.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
```

Comment:
This file probably isn't doing anything right now. If it isn't breaking anything then keeping it should be fine.

Maybe add a task to asana to create a storybook item for the crossing list?